mscore/musescore.cpp

```diff
@@ -4865,6 +4879,8 @@ void MuseScore::cmd(QAction* a, const QString& cmd)
             if (mixer)
                   mixer->updateAll(cs->masterScore());
             }
+      else if (cmd == "minimize-window")
+            MuseScoreApplication::activeWindow()->showMinimized();
```

Comment:
Isn't that Mac-only?

Reply:
Why would minimize window be Mac specific? I'm sure Windows and Linux have that concept too.

Reply:
@Jojo-Schmitz I'm confused as to exactly what you're asking. If I understand you correctly, you're asking why the Q_OS_MAC exists in the other file.

It exists, not to restrict the use of the shortcut on other platforms (which may already have a working, built-in minimize shortcut), but to refrain from adding what I believe is a "traditionally Mac" Window menu that contains the Minimize option.